index.js

```diff
@@ -13,12 +13,136 @@ mongoose
   .then(x => {
     console.log(`Connected to the database: "${x.connection.name}"`);
     // Before adding any recipes to the database, let's remove all existing ones
-    return Recipe.deleteMany()
+    return Recipe.deleteMany();
   })
-  .then(() => {
+  console.log(Recipe.title)
+  .then(() =>  {
```

Comment:
You cannot have any code in between `.then()`

think of it as connected chain:

`mongoose.connect().then().then().then() `etc

But now you added console.log which will break your code

`mongoose.connect().then()consolelog().then().then() `